layouts/event/single.html

```diff
@@ -30,6 +30,11 @@
             {{ with .Params.duration }}<div>
                 <strong>{{ T "event-duration" }}</strong><br> {{ . }}
             </div>{{ end }}
+            <br>{{ T "blog-licensed-under" }} <a
```

Comment:
I don't think that this text is fitting here. Something short like "License" would be much better. You can add this to the translations file in `src/i18n/en.json` in the `hugo` section.

Reply:
I think hugo section already contains a translation on line number 111:
<img width="415" alt="Screenshot 2020-10-02 at 4 24 04 PM" src="https://user-images.githubusercontent.com/53044263/94916138-bd8def80-04cb-11eb-8835-b2913b70daba.png">

Reply:
Yes, exactly. But this translation is meant to be placed at the end of a blog post, not as a label at the top under a video. What I was suggesting was adding a new string, e.g. "License", that would be a better label in the place where you put it :)